bottlecap/tests/metrics_integration_test.rs

```diff
@@ -0,0 +1,64 @@
+use bottlecap::config::Config;
+use bottlecap::metrics::aggregator::Aggregator as MetricsAggregator;
+use bottlecap::metrics::enhanced::lambda::Lambda as enhanced_metrics;
+use bottlecap::metrics::flusher::Flusher as MetricsFlusher;
+use bottlecap::tags::provider::Provider;
+use bottlecap::LAMBDA_RUNTIME_SLUG;
+use httpmock::prelude::*;
+use std::collections::HashMap;
+use std::sync::{Arc, Mutex};
+
+mod common;
+
+#[tokio::test]
+async fn test_enhanced_metrics() {
+    let dd_api_key = "my_test_key";
+
+    // payload looks like
+    // aws.lambda.enhanced.invocations"_dd.compute_stats:1"architecture:x86_64"function_arn:test-arn:�൴      �?!      �?)      �?1      �?:�B
```

Comment:
got some non unicode symbols here

Reply:
I didn't want to spend too much time trying to output the raw string there. I am matching the readable parts only